scripts/prepare-release.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+PREVIOUS_VERSION="$(cat deploy/olm-catalog/integreatly-operator/integreatly-operator.package.yaml | grep integreatly-operator | awk -F v '{print $2}')"
+
+create_new_csv() {
+  operator-sdk generate csv --csv-version "$VERSION" --default-channel --csv-channel=rhmi --update-crds --from-version "$PREVIOUS_VERSION"
+}
+
+# Creating a new release version
+set_new_version() {
+  sed -i "s/image:.*/image: quay\.io\/integreatly\/integreatly-operator:v$VERSION/g" deploy/operator.yaml
+  sed -i "s/$PREVIOUS_VERSION/$VERSION/g" version/version.go
+  sed -i "s/$PREVIOUS_VERSION/$VERSION/g" Makefile
+  sed -i "s/PREVIOUS_TAG ?=.*/PREVIOUS_TAG ?=$PREVIOUS_VERSION/g" Makefile
+  sed -i "s/image:.*/image: quay\.io\/integreatly\/integreatly-operator:v$VERSION/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
+  sed -i "s/containerImage:.*/containerImage: quay\.io\/integreatly\/integreatly-operator:v$VERSION/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
+}
+
+# Creating a new tag on the already existing release
+set_new_tag() {
+  sed -i "s/image:.*/image: quay\.io\/integreatly\/integreatly-operator:v$PREVIOUS_VERSION-$TAG/g" deploy/operator.yaml
+  sed -i "s/image:.*/image: quay\.io\/integreatly\/integreatly-operator:v$PREVIOUS_VERSION-$TAG/g" deploy/olm-catalog/integreatly-operator/${PREVIOUS_VERSION}/integreatly-operator.v${PREVIOUS_VERSION}.clusterserviceversion.yaml
+  sed -i "s/version: ${PREVIOUS_VERSION}.*/version: ${PREVIOUS_VERSION}-${TAG}/g" deploy/olm-catalog/integreatly-operator/${PREVIOUS_VERSION}/integreatly-operator.v${PREVIOUS_VERSION}.clusterserviceversion.yaml
+}
+
+# Creating a new release with a tag
+set_new_version_with_tag() {
+  sed -i "s/$PREVIOUS_VERSION/$VERSION/g" Makefile
+  sed -i "s/PREVIOUS_TAG ?=.*/PREVIOUS_TAG ?=$PREVIOUS_VERSION/g" Makefile
+  sed -i "s/image:.*/image: quay\.io\/integreatly\/integreatly-operator:v$VERSION-$TAG/g" deploy/operator.yaml
+  sed -i "s/containerImage:.*/containerImage: quay\.io\/integreatly\/integreatly-operator:v$VERSION/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
+  sed -i "s/image:.*/image: quay\.io\/integreatly\/integreatly-operator:v$VERSION-$TAG/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
+  sed -i "s/version: $VERSION.*/version: $VERSION-$TAG/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
+}
+
+clean_up() {
+  rm -rf deploy/olm-catalog/integreatly-operator/${PREVIOUS_VERSION}
+}
```

Comment:
After talking to @philbrookes i don't think we should do this any more. 2.1.0+ will be setting the replaces and will need the previous versions kept.